Scripts/interactable/NumberLogic/MemoryPanel.lua

```diff
@@ -127,6 +144,26 @@ end
 function MemoryPanel.client_onCreate(self)
 	self.mode = 0
 	self.time = 0
+
+	-- Create GUI
+	if not mem_gui then
+		mem_gui = sm.gui.createGuiFromLayout("$CONTENT_DATA/Gui/Layouts/MemoryPanelGui.layout", false, { backgroundAlpha = 0.5 })
+
+		mem_gui:setButtonCallback("SaveWrittenVal", "client_gui_saveWrittenValue")
+
+		mem_gui:setTextChangedCallback("ValueInput", "client_onTextChangedCallback")
+		mem_gui:setOnCloseCallback("client_onGuiCloseCallback")
```

Comment:
Small nitpick, removed the empty lines
```suggestion
		mem_gui = sm.gui.createGuiFromLayout("$CONTENT_DATA/Gui/Layouts/MemoryPanelGui.layout", false, { backgroundAlpha = 0.5 })
		mem_gui:setButtonCallback("SaveWrittenVal", "client_gui_saveWrittenValue")
		mem_gui:setTextChangedCallback("ValueInput", "client_onTextChangedCallback")
		mem_gui:setOnCloseCallback("client_onGuiCloseCallback")
```